xml/System.Numerics/BitOperations.xml

```diff
@@ -78,8 +78,8 @@
         <Parameter Name="value" Type="System.Int64" Index="0" FrameworkAlternate="net-6.0" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="value">The value.</param>
+        <summary>Evaluate whether a given integral value is a power of 2.</summary>
         <returns>To be added.</returns>
```

Comment:
```suggestion
        <returns><see langword="true" /> if the given integral value is a power of 2; <see langword="false" /> otherwise.</returns>
```